catroid/src/main/res/values-pl/strings.xml

```diff
@@ -1802,7 +1802,7 @@ Jeśli jednak nie chcesz tego robić, możesz skorzystać z naszych bezpłatnych
     <string name="formula_editor_logic_comparison">operatory porównania</string>
     <string name="formula_editor_object_general">ustawienia ogólne</string>
     <string name="formula_editor_object_movement">właściwości ruchu</string>
-    <string name="formula_editor_device_visual_detection">Czujniki widzenia</string>
+    <string name="formula_editor_device_face_detection">Czujniki widzenia</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations